pennylane/qnn/keras.py

```diff
@@ -232,14 +232,7 @@ def __init__(
         else:
             self.qnode = batch_input(qnode, argnum=batch_idx)
 
-        dtype = tf.float32 if tf.keras.backend.floatx() == tf.float32 else tf.float64
```

Comment:
This is also not needed? Because we now pass types through the qnode?

Reply:
Actually, this is not needed just because the new pipeline does not support `float32` yet 😬 This is something we still need to address, but we need to do it at a more fundamental level.

Basically, we should ensure that all of our simulators support multiple types, and then make the core type agnostic